README.md

```diff
@@ -1,31 +1,23 @@
-|Gitter|Build|Links|
-|------|-----|-----|
-|[![Gitter](https://badges.gitter.im/Galaxy-Training-Network/training-material.svg)](https://gitter.im/Galaxy-Training-Network/Lobby?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge) |[![Build Status](https://travis-ci.com/galaxyproject/training-material.svg?branch=master)](https://travis-ci.com/galaxyproject/training-material) | [![Deploy Status](https://travis-ci.com/galaxyproject/training-material.svg?branch=gh-pages)](https://travis-ci.com/galaxyproject/training-material)|
+![GTN logo, a star on a coloured circle with the text Galaxy Training Network at an upward angle.](./shared/images/GTNLogo1000.png)
```

Comment:
the image renders really big, or is this just the preview?

Reply:
no, it's quite big.